libats-http/src/main/scala/com/advancedtelematic/libats/http/ServiceHttpClient.scala

```diff
@@ -18,6 +18,7 @@ import scala.concurrent.Future
 import scala.reflect.ClassTag
 import scala.util.Try
 
+case class UnmarshalledHttpResponse[T : FromEntityUnmarshaller](httpResponse: HttpResponse, unmarshalledResponse: T)
```

Comment:
Why do you need `: FromEntityUnmarshaller` here?

Reply:
Because `execHttp` and `execHttp2` require an implicit `FromEntityUnmarshaller[T]` in scope.

Reply:
So I guess it's not _actually_ needed, because if the type class is not available `execHttp` will not compile. But since `UnmarshalledHttpResponse` is meant to be used in this way we might as well restrict it there too...

Reply:
Yeah but that doesn't matter anymore for `UnmarshalledHttpResponse`, no? It is technically true that it came from something unmarshalled, but is that really important when you use it?

Reply:
Ah I see your point now. Yes, it makes sense to me. I will remove that constraint.

Reply:
Done.